src/components/chips/contact-chips.spec.js

```diff
@@ -82,6 +83,15 @@ describe('<md-contact-chips>', function() {
         expect(chip.find('img').length).toBe(0);
     });
 
+    it('check if minLength on autocomplete controller equals md-min-length attribute on md-contact-chips element', inject(function() {
```

Comment:
Can we rename this to `should forward md-min-length attribute to the autocomplete` ?